publishable/lang/fr/generic.php

```diff
@@ -87,4 +87,5 @@
     'viewing'                => 'Affichage',
     'yes'                    => 'Oui',
     'yes_please'             => 'Oui SVP',
+    'is_translatable'        => 'Translatate?',
```

Comment:
`'is_translatable'        => 'Traduire?',`

Reply:
i just add the necessary keys for all language files in voyager on english, after that, people will translate the key for the correct sentence in his language.

Reply:
Yes okay.